packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-view.tsx

```diff
@@ -9,16 +9,31 @@ export interface MatchViewMatch {
 	firstTeam: Team;
 	secondTeam: Team;
 	winner: Team | null;
+	contested: boolean;
 }
 
 interface Props {
 	reportWin: (teamId: string) => void;
+	reportContested: (contested: boolean) => void;
 	match: MatchViewMatch;
 }
 
-const MatchView: React.FC<Props> = ({ reportWin, match }) => {
+const MatchView: React.FC<Props> = ({ reportWin, reportContested, match }) => {
 	if (match.winner) {
-		return <h1>The winner is: {match.winner.name}</h1>;
+		if (match.winner && match.contested) {
+			return (
+				<div>
+					<h1>The winner is: {match.winner.name}</h1>
+					{match.contested && <h1>The match has been contested!</h1>}
+					{/*	 idk, if I dont have an arrow function in the onclick gets sad */}
+					{!match.contested && (
+						<button onClick={() => reportContested(true)}>Contest result!</button>
```

Comment:
onClick expected a function that has one parameter of type React.ClickEvent<HTMLButtonElement> (ish), but the first parameter of of reportContested is of type boolean

Reply:
Yeah it expected like ((clickEventReactSomething => Void) | undefined) so its like I get it but it kinda lmao